assets/src/blocks/Splittwocolumns/imageSizes.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  columnLeft: `
+  (min-width: 580px) 80vw,
+  100vw
+  `,
+  columnRight: `
+  (min-width: 580px) 75vw,
+  1vw
```

Comment:
The right column is hidden under `580px`, so actually the image is not needed in that case. But the browser still loads it, and I didn't see an easy way to prevent it totally. I tried using `0` and `0vw`, but it still tried to load it. Setting it to `1vw` at least only makes it load the smallest possible image. Could be I'm overlooking a much better solution.

Reply:
Tried with lazy loading (which we should add anyway I think ?), still doesn't work :/ From [the old stuff](https://timkadlec.com/2012/04/media-query-asset-downloading-results/) I'm reading, the solutions would be to put the image as a css background, or condition the img tag insertion to the block width using [`useEffect`](https://stackoverflow.com/a/56011277)

Reply:
Maybe we can review the block's specs first, since it's a bit weird that the right column is simply hidden below a certain screen size, I think most editors don't expect or realize this. Both columns seem equivalent in importance.

Reply:
Lazy loading wouldn't help for this, as that only checks whether a given element is within or close to the viewport, not whether it's actually visible. We can still add it, though maybe better after we make the block static.